scripts/run_integration_tests_x86_64.sh

```diff
@@ -8,6 +8,45 @@ export BUILD_TARGET=${BUILD_TARGET-x86_64-unknown-linux-gnu}
 WORKLOADS_DIR="$HOME/workloads"
 mkdir -p "$WORKLOADS_DIR"
 
+hypervisor="kvm"
+cmd_help() {
+    echo ""
+    echo "Cloud Hypervisor $(basename $0)"
+    echo "Usage: $(basename $0) [<args>]"
+    echo ""
+    echo "Available arguments:"
+    echo ""
+    echo "    --hypervisor  Underlying hypervisor. Options kvm, mshv"
+    echo ""
+    echo "    --help        Display this help message."
+    echo ""
+}
+process_args() {
```

Comment:
This is getting rather repetitive. Can you extract the common parameter parsing code (name it `process_common_args`) to something like `test-util.sh` and then source that file in all the test scripts?

Reply:
Yes. I was planning to do that. Thanks anyway.